test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
no problem :-)

Two.) `render json: Blog.first, include: 'posts'`
```
data: 
  blog
included:
  posts
  authors
```

Three.) `render json: Blog.first, include: 'posts.category'`
```
data: 
  blog
included:
  posts
  authors
  categories
```

So, the include option is only merged with the default includes in the top-most serializer, in my perfect world, at least :-)